packages/buckets/src/buckets.ts

```diff
@@ -660,7 +662,15 @@ export class Buckets extends GrpcAuthentication {
     input: any,
     options?: PushOptions,
   ): Promise<PushPathResult> {
-    return bucketsPushPathNode(this, key, path, input, options)
+    return bucketsPushPath(this, key, path, input, options)
```

Comment:
Dropping the "node" version in favor of just the one, which _is_ actually the "node" style emitter version.